integration-test/tests/cms/team.test.ts

```diff
@@ -0,0 +1,12 @@
+import {expect, test} from '@playwright/test';
+
+test('GET /api/cms/v1/team returns correct data', async ({request}) => {
+    const response = await request.get('/api/cms/v1/team');
+    expect(response.status()).toBe(200);
+    const data = await response.json();
+    expect(response).toBeDefined();
+    expect(data.page).toBeDefined();
+    expect(data.page.title).toBe('Team');
+    expect(data.contact).toBeDefined();
+    expect(data.contact.title).toBe('Contact us');  
```

Comment:
This validation will cover API contract validation only partially, since it will check only root properties and will not go deep inside JSON structure. I would highly recommend to use schema verification for checking the whole JSON tree.

Reply:
Hi, do I need to add team.data.ts file as well, as I was thinking the inclusion of actual response data in the data .ts file contains personal data?

Reply:
It is better to verify schema rather than response itself. Two reasons: personal information, team info can change frequently in the future.

Reply:
+1 to what Nora mentioned!

Reply:
Please, also refer to the latest PR merged, it will help to separate some common schemas and reuse them. 
https://github.com/Women-Coding-Community/wcc-backend/pull/164